Tests/Fakes/FakeCollectionViewModel.swift

```diff
@@ -20,16 +20,29 @@ extension XCTestCase {
     @MainActor
     func fakeCollectionViewModel(
         id: String = .random,
+        sectionId: (Int) -> String = { _ in .random },
+        cellId: (Int, Int) -> String = { _, _ in .random },
+        supplementaryViewId: (Int, Int) -> String = { _, _ in .random },
```

Comment:
I think the issue here is that some of the fake models have ids that are `Int` correct?

I think we should simplify and just make all ids `String` so then we can just pass strings here.

Or am I missing something? These closures just feel too complex to me.

Reply:
The current `id` is a random `UUID`, which cannot be compared with the given `debugDescription`, so the `id` is changed to be initialized by a closure.

Reply:
1. `sectionId: (Int) -> String = { _ in .random }` -> `sectionId: (Int) -> String = { sectionIndex in .random }`
2. `cellId: (Int, Int) -> String = { _, _ in .random }` -> `cellId: (Int, Int) -> String = { sectionIndex, cellIndex in .random }`
3. `supplementaryViewId: (Int, Int) -> String = { _, _ in .random }` -> `supplementaryViewId: (Int, Int) -> String = { sectionIndex, cellIndex in .random }`